### PR TITLE
Fix search bar error for members

### PR DESCRIPTION
**Pull request by airaagrawal1**
Co-authored-by: Henry Li <hl738@cornell.edu>

### Summary 

This fixed the issue where the search bar for members was not auto-populating search results. The issue was resolved by checking that the field for a member was not null before trying to match it to what was being searched. 

### Test Plan 

Tested on local machine and was able to find a member and add/delete them from a team.

### CODE REVIEW

**Comment by JacksonStaniec** [comment]
It looks like Prettier didn't run. For some reason Prettier wasn't formatting on save for me, so just alt+shift+F to format (or whatever your shortcut is) for each file.

**Review by JacksonStaniec — COMMENTED** [review]
Besides the formatting everything looks good!

**Comment by airaagrawal1** [comment]
Fixed the formatting, thanks!

File: frontend/src/components/Admin/EditTeam.tsx [hunk 1]
```diff
@@ -219,13 +219,17 @@ class EditTeam extends React.Component<Record<string, unknown>, EditTeamState> {
...
+                            (member.firstName &&
+                              member.firstName.toLowerCase().startsWith(queryLower)) ||
+                            (member.lastName &&
+                              member.lastName.toLowerCase().startsWith(queryLower)) ||
+                            (member.role && member.role.toLowerCase().startsWith(queryLower)) ||
+                            ((member.firstName &&
+                              member.lastName &&
+                              `${member.firstName.toLowerCase()} ${member.lastName.toLowerCase()}`.startsWith(
+                                queryLower
+                              )) as boolean)
```
Comment (SamChou19815) [inline frontend/src/components/Admin/EditTeam.tsx:232]: While you are here, let's replace the copy-pasta with one common checker.

Comment (airaagrawal1) [reply frontend/src/components/Admin/EditTeam.tsx:232]: Was running into some issues while trying to refactor the code. Can I make a separate PR for refactoring the code?

**Review by SamChou19815 — APPROVED** [review]

**Comment by riyajaggi** [comment]
> It looks like Prettier didn't run. For some reason Prettier wasn't formatting on save for me, so just alt+shift+F to format (or whatever your shortcut is) for each file.

For the next pr, you can try running the command `yarn format` and then commit your changes. This command should fix the style issues for all files.

**Pull request merged**